index.html

```diff
@@ -279,5 +279,34 @@ <h3 class="titleA">About My self</h3>
                 </li>
               </ul>
             </section>
-  
-  
\ No newline at end of file
+            <!--Works Section contact page-->
+            <section class="page5">
+              <div class="@continer">
+                <div class="bag">
+               
+                <div class="h1">
+                  <form action="https://formspree.io/f/mzbqngjn">
```

Comment:
- [ ] According to the project requirement your form should submit without any issue. To fix this issue please implement `method="post"` and also to send the form normally please use the `name` attribute in your input field. Then the form will send normally.
### Error now
![his](https://github.com/sadat-Ramin/My-portfolio/assets/81727988/5999059d-bba9-4309-9c8a-4e8653a07fc6)